LIPs/LIP-25.md

```diff
@@ -0,0 +1,237 @@
+
+### Upgrade Path
+
+The governance contract's design rationale should establish a clear technical foundation to lay the groundwork for an upgrade path to give control over protocol code/parameters to the stakeholders. This section provides an example of what such an upgrade path might look like without setting explicit milestones. 
+
+1. Deploy governance contract with the core team multisig as `owner`. 
+2. Make core team multisig subject to a time delay for most actions (e.g. it might still be interesting to allow the core team to retain the right to pause/unpause the protocol in the early stages)
+3. Grant access to a binding voting system to alter less sensitive protocol parameters
+4. Hand over control of all protocol parameters to binding voting systems
+5. Allow binding voting systems to execute code updates
+6. Deploy a binding voting system to govern the governance contract's ACL
+7. Remove core team multisig as `owner` and have ACL rules be established by a binding voting system
+
+**In scope** For this proposal would be **step 1** and **step 2**
+
+Step 1 would be completed upon deployment of the Extensible Governance System. Step 2 will be decided by the outcome of the initial values discussion [here](https://github.com/livepeer/LIPs/issues/30). 
+
+
+## Specification Rationale
+
+A modular ACL list that is actor agnostic allows for maximum extensibility to the governance structure and abstracts any details of how a binding voting systems should look like. It merely sets the foundation through which different actors can execute updates to protocol code or parameters. 
+
+It further enables different types of actors or different types of voting systems to have control over different parts of the protocol. Different parameters might require different systems, e.g. higher `QUORUM` and `THRESHOLD` values or  different voting schemes than tokenholder voting.
+
+Facilitating executing batches of staged updates simplifies the upgrade path for both EOA and contract actors and provides full transparency to all stakeholders as to all staged updates, their actors and time delay. 
+
+A modular ACL and time delay allows for interesting mechanics and a clear governance upgrade path to hand control over to binding voting systems. The core team can initially retain `owner` access over the protocol contracts but updates subject to the LIP process can already be subject to a time delay as well. In future milestones control over certain parameters or code updates can be phased over to the community through a binding voting system. 
+
+## Initial Parameter Values
+
+There's mainly two initial values that need to be set 
+
+- The initial _master_ actor 
+- The `DELAY` for the first actor, if applicable 
+
+It's established already that the first actor of this governance system will be the Livepeer Inc Multisig. Additional actors can be added over time through the Livepeer Governance process. 
```

Comment:
The address of this multisig should be included in the document if it is one of the default parameter values.

Reply:
renamed to `owner` and added the address in 1afd126